environment.yml

```
@@ -56,5 +56,6 @@ dependencies:
   - pytest-rerunfailures
   - pytest-timeout
   - pytest-xdist
+  - pytest-xvfb
```

Probably this is the easiest way forward
```suggestion
  # - pytest-xvfb  # can be included on linux/win to suppress windows popping up during the test
```